docs/wegue-configuration.md

```diff
@@ -24,6 +24,56 @@ This describes the Wegue application configuration, which is modelled as JSON do
 | projectionDefs     | Array of CRS / projection definition objects compatible to proj4js | See [projectionDefs](wegue-configuration?id=projectiondefs) |
 | tileGridDefs       | Array of tile grid definition objects | See [tileGridDefs](wegue-configuration?id=tilegriddefs) |
 
+### theme
+
+Wegue supports the Vuetify (light and dark) theme configuration out of the box. This property expects a json object with the same format described in the [vuetify documentation](https://vuetifyjs.com/en/features/theme/#customizing).
+
+Example:
+```json
```

Comment:
are all 7 colors per theme compulsory?

Reply:
To fully colorize the app we need the `primary`, `secondary` and `accent`. The remaining are occasionally needed in case we set a light background with white text.

I'll include that detail in the documentation too.

Reply:
Hi @JakobMiksch,

In the recent changes, only the "primary" color is mandatory for the "light" theme and the "secondary" for the "dark" theme. The reason between this asymmetry is explained in the [docs](https://github.com/meggsimum/wegue/pull/262/files#diff-c012d313a14a5074d6d5d8d996f6c0e4188266068540fab41390c2200d1282ecR95).

Feel free to reopen this review if needed :+1: